frontend/ui/widget/filechooser.lua

```diff
@@ -206,6 +206,44 @@ local FileChooser = Menu:extend{
                 return item.opened and string.format("%d %%", 100 * item.percent_finished) or "–"
             end,
         },
+        percent_natural = {
+            -- sort 90% > 50% > 0% > unopened > 100%
+            text = _("percent - unopened last natural"),
```

Comment:
Simply descriptive in this direction?
```suggestion
            text = _("percent - unopened - finished last"),
```